pkg/report/diffcoverage.go

```
@@ -31,12 +34,23 @@ func NewDiffCoverage(
 		excludesRegexps = append(excludesRegexps, reg)
 	}
 
+	ignoreProfiles := make(map[string]*annotation.IgnoreProfile)
+	for _, c := range changes {
+		ignoreProfile, err := annotation.ParseIgnoreProfiles(filepath.Join(repositoryPath, c.FileName))
+		if err == nil {
+			ignoreProfiles[c.FileName] = ignoreProfile
+		} else {
+			fmt.Printf("warn: %s\n", err)
```

why this is a warning?

---

I think we should fail fast